google/cloud/bigquery_storage.py

```diff
@@ -17,9 +17,9 @@
 
 from __future__ import absolute_import
 
-from google.cloud.bigquery_storage_v1beta1 import BigQueryStorageClient
-from google.cloud.bigquery_storage_v1beta1 import enums
-from google.cloud.bigquery_storage_v1beta1 import types
+from google.cloud.bigquery_storage_v1 import BigQueryReadClient
```

Comment:
@tswast This is one change I'm worried about.  Do we rely on this non-versioned client for anything elsewhere that would care about getting shifted from v1beta1 to v1?  It's going to require further changes to address if so.

Reply:
In our public samples, I always used the `v1beta1` module. There are a few versions of `pandas-gbq` that use the versionless module, but I fixed that in the latest version. (https://github.com/pydata/pandas-gbq/commit/e177978227c9a42e3c63a864616c6ac7d98f2840#diff-4db670026d33c02e5ad3dfbd5e4fd595R11)

So, yes: this will break a few versions of `pandas-gbq`. Might be worth making some aliases for `BigQueryStorageClient` and the necessary methods to avoid breakages.

Reply:
Given the proto messages have changed between v1beta1 and v1beta2 it seems you'd need more than just an alias to get the right client name.  Things like tablereferences became a single string rather than a structured object, sharding strategy went away (everything is balanced), which makes converting a bit dicey if you were expecting the old strategy.

Reply:
Gotcha. When customers hit this, we can tell them to either upgrade pandas-gbq or pin their google-cloud-bigquery-storage version.